packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.module.scss

```diff
@@ -0,0 +1,20 @@
+:local {
+  iframe {
+    border: 0;
+    width: 100%;
+  }
```

Comment:
Before this, this isn't part of it. Let's remove this iframe style since it's up to consumers how they want the iframe to look.

Reply:
Removed here: https://github.com/cerner/terra-framework/pull/1804/commits/9b1fbec0ce611cf82484cd3eec8a97d8afc53971